templates/app.yaml

```diff
@@ -416,6 +416,19 @@ Resources:
         - !ImportValue
           'Fn::Sub': '${AWS::Region}-sc-kms-key-KmsDecryptPolicyArn'
         - !Ref SsmManagedPolicy
+        - !ImportValue
+          'Fn::Sub': '${AWS::Region}-marketplace-dynamo-MarketplaceDynamoDBTableArn'
+      Policies:
```

Comment:
an AWS best practice is to define and use `managed` policies instead of `inline` policies.  please use the same pattern as `SsmManagedPolicy`.

Reply:
Done!